### ISSUE TITLE
Adding a keybind button for start

### ISSUE THREAD

**Issue by gituser98654** [jagt/clumsy#81]
Hello, I'd like to ask if it would be possible to add a keybind option for the start button. Thank you very much, I appreciate your work.

**Comment by Kacperowsky** [comment]
Yes, you can add keybind. You can simply do it with AutoHotkey, here is the script:

L::ControlClick, Button2, clumsy 0.2

L is the keybind, you can change it whatever you want, like F.

Reply if i helped.

**Comment by gituser98654** [comment]
I already found an ahk script for it but thanks anyways.

**Comment by Kacperowsky** [comment]
no problem

**Comment by WindoPvP** [comment]
Can you give me your ahk script ? @gituser98654

**Comment by DelinquentX** [comment]
can i get the ahk script??
